contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,303 @@ availability:
-We can see recommendations for run times and other experiment factors.
+By default, your experiment will target 100% of participants. If you'd like to target a more specific set of participants, or change rollout percentage, you'll need to do this on the feature flag that will be used by your experiment.
 
-![Experiment preview](../../images/features/experiments/create-experiment-preview.png)
+Below is a video showing this:
 
-We can also increase or decrease our minimal acceptable improvement depending on how much we care for our test's significance relevant to our other experiment factors.
+// TODO Add video of how to do this
 
-Perhaps we don't want to wait 278 days for our experiment to run and we'd prefer less precise results in favor of a shorter run time. We can simply bump the threshold to a higher percentage.
+### Experiment goal
 
-![Experiment preview significance change](../../images/features/experiments/create-experiment-preview-significance.png)
+Setting your goal metric will enable PostHog to automatically keep track of your goal metrics and let you know if your results are statiscally signifcant. You can select between a "trend" or "conversion funnel" goal.
 
-Whenever we're ready, hit save on the bottom right corner of the form to create an experiment. Creating an experiment saves it into a draft mode so we'll still have flexibility on making adjustments to it until we're ready to launch it!
+A minimum acceptable improvement calculator is only shown on the screen. This combines historical data from your goal metric with your desired improvement rate to show you how long you'll need to run your experiment for in order to see statiscal significant results.
 
-## Running an experiment
+![Minimum acceptable improvement calculator](../../images/features/experiments/improvement-calculator.mp4)
 
-Before you can launch an experiment, you need to implement the code for each feature flag variant. On the draft experiment screen, you'll see a snippet telling you how to query for the variant. It looks something like:
+> **Note:** When selecting an event that is submitted from your server, you may see an warning that no feature flag information can be detected with the event. To resolve this issue, see [step 2 of adding your experiment code and how to submit feature flag information with your events](/docs/experiments/manual#step-2-server-side-only-upload-the-feature-flag-property-with-your-events).
+> 
+> ![Experiment goal warning](../../images/features/experiments/experiment-goal-warning.png)
 
-```js
-if (posthog.getFeatureFlag('your-chosen-feature-flag-key') === 'control') {
-    // where 'control' is the variant, run your code here
+## Adding your experiment code
+
+### Step 1: Fetch the feature flag
+
+To check which variant of your experiment to show to your participants, use the following code to fetch the feature flag:
+
+<MultiLanguage>
```

Comment:
Maybe you can split this into frontend vs backend and mention the need for distinct_id on the backend? And how users get automatically split.